packages/free/src/adapter.ts

```diff
@@ -25,11 +25,11 @@ class Storage {
 
   async call(
     method: 'GET' | 'POST' | 'DELETE',
-    key: string,
+    key?: string, // make key optional to allow global calls
     body?: string,
   ): Promise<string | undefined> {
     // perform request
-    const url = `${this.rootUrl}/session/${key}`;
+    const url = key ? `${this.rootUrl}/session/${key}` : `${this.rootUrl}/sessions`; // define url based on key
```

Comment:
You introduced a bug. Checking for truthiness is evil. What if the empty session key is used?
```suggestion
    const url = key !== undefined ? `${this.rootUrl}/session/${key}` : `${this.rootUrl}/sessions`; // define url based on key
```

Reply:
yes yes indeed if key = "", we will call /sessions instead of /session/.
Thank you I'll adjust that.